packages/@o3r/core/builders/run-script/index.ts

```diff
@@ -0,0 +1,37 @@
+
+import { BuilderOutput, createBuilder } from '@angular-devkit/architect';
+import { getPackageManagerRunner } from '@o3r/schematics';
+import { execSync } from 'node:child_process';
+import { join, resolve } from 'node:path';
+import * as fs from 'node:fs';
+import type { RunScriptBuilderSchema } from './schema';
+
+export default createBuilder<RunScriptBuilderSchema>(async (options, context): Promise<BuilderOutput> => {
+  context.reportRunning();
+  context.reportProgress(1, 1, `Running ${options.script} !!!`);
+  const specifiedRoot = context.target?.project && (await context.getProjectMetadata(context.target.project)).root?.toString();
+  const projectRoot = specifiedRoot ? resolve(context.workspaceRoot, specifiedRoot) : context.currentDirectory;
+
+  const angularJsonPath = join(context.workspaceRoot, 'angular.json');
+  let angularJson;
```

Comment:
not sure why the linter did not complain regarding the implicit `any`.
Can be written:
```typescript
const angularJson = fs.existsSync(angularJsonPath) ? JSON.parse(fs.readFileSync(angularJsonPath, {encoding: 'utf8'}) : undefined;
if (!angularJson) {
  context.logger.warn(`angular.json file cannot be found by @o3r/core:${context.builder.builderName} builder.
Detection of package manager runner will fallback on the one used to execute the actual command.`);
}
```

Reply:
updated. I guess we have no error because `noImplicitAny` compiler option is true by default.